ComponentKit/Utilities/CKComponentAction.mm

```diff
@@ -67,6 +67,27 @@ void CKConfigureInvocationWithArguments(NSInvocation *invocation, NSInteger inde
                   NSStringFromSelector(selector), _CKComponentResponderChainDebugResponderChain(target));
   // This is not performance-sensitive, so we can just use an invocation here.
   NSMethodSignature *signature = [responder methodSignatureForSelector:selector];
+  while (!signature) {
+    // From https://www.mikeash.com/pyblog/friday-qa-2009-03-27-objective-c-message-forwarding.html
+    // 1. Lazy method resolution
+    [[responder class] resolveInstanceMethod:selector];
```

Comment:
Do we care about the return value here or are we just relying on getting a signature immediately below? Would being explicit be better?